pkg/controller/nodes/task/backoff/handler.go

```diff
@@ -24,33 +26,40 @@ var (
 type SimpleBackOffBlocker struct {
 	Clock              clock.Clock
 	BackOffBaseSecond  int
-	BackOffExponent    int
-	NextEligibleTime   time.Time
 	MaxBackOffDuration time.Duration
+
+	// Mutable fields
+	BackOffExponent  stdAtomic.Uint32
+	NextEligibleTime atomic.Value
```

Comment:
Ideally we should implement the same interface as the stdAtomic for time, if we can
`NextEligibleTime atomic.Value` Then we can migrate it to stdlib at somepoint